src/main/java/net/neoforged/neoforge/transfer/handlers/templates/contexts/CreativePlayerItemContext.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.transfer.handlers.templates.contexts;
+
+import net.minecraft.world.entity.player.Player;
+import net.neoforged.neoforge.transfer.ResourceHandlerUtil;
+import net.neoforged.neoforge.transfer.handlers.wrappers.items.PlayerInventoryWrapper;
+import net.neoforged.neoforge.transfer.resources.ItemResource;
+import net.neoforged.neoforge.transfer.transaction.TransactionContext;
+
+public class CreativePlayerItemContext extends StaticItemContext {
```

Comment:
There should be a comment *somewhere* as to why this doesn't just extend `PlayerItemContext`